flask_lesson_2/config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+class Config(object):
```

Comment:
```suggestion
class Config:
```
в 3 пайтоні не обов'язково посилатися на базовий клас при створенні нового

Reply:
ок